patches/server/0720-More-Projectile-API.patch

```diff
@@ -559,6 +568,35 @@ index 25133298b26523bd36300ab03e216200c915386b..af7e3acde2eb521ac809b4c5980ff6ad
              hitEntity = ((EntityHitResult) position).getEntity().getBukkitEntity();
          }
  
+@@ -875,19 +875,21 @@ public class CraftEventFactory {
+     }
+ 
+     // Paper start - Fix potions splash events
+-    public static io.papermc.paper.event.entity.WaterBottleSplashEvent callWaterBottleSplashEvent(net.minecraft.world.entity.projectile.ThrownPotion potion, HitResult hitResult, Map<LivingEntity, Double> affectedEntities, java.util.Set<LivingEntity> rehydrate, java.util.Set<LivingEntity> extinguish) {
++    public static io.papermc.paper.event.entity.WaterBottleSplashEvent callWaterBottleSplashEvent(net.minecraft.world.entity.projectile.ThrownPotion potion, @Nullable HitResult hitResult, Map<LivingEntity, Double> affectedEntities, java.util.Set<LivingEntity> rehydrate, java.util.Set<LivingEntity> extinguish) {
```

Comment:
I'd just make this nullable in the patch the method is added to avoid diff over diff